libnetdata/ebpf/ebpf.c

```diff
@@ -4,6 +4,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <dlfcn.h>
+#include <linux/version.h>
```

Comment:
This header is present only when we install Linux headers, with the code we have on this PR I do not think it is necessary.

Reply:
I saw you are using `KERNEL_VERSION`, I will suggest you use another definition like `NETDATA_MINIMUM_EBPF_KERNEL` to avoid bring this hard dependency. Or is it required now with `libbpf`?

Reply:
You are right. I've removed the dependency.